src/main/java/tr/edu/ogu/ceng/service/StudentManager.java

```diff
@@ -0,0 +1,31 @@
+package tr.edu.ogu.ceng.service;
+
+import java.sql.Timestamp;
+
+import org.springframework.stereotype.Service;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import tr.edu.ogu.ceng.dao.StudentRepository;
+import tr.edu.ogu.ceng.dto.requests.UpdateStudentRequest;
+import tr.edu.ogu.ceng.dto.responses.UpdateStudentResponse;
+import tr.edu.ogu.ceng.model.Student;
+import tr.edu.ogu.ceng.utilities.mappers.*;
+
+@Service
+@AllArgsConstructor
+public class StudentManager implements StudentService{
```

Comment:
Servis yapısı için diğer PR'ları incelemeniz de fayda var, ayrı bir interface sonra da implement eden bir sınıf oluşturulmadı hiç.